winui/Rating/Precision.md

```diff
@@ -1,12 +1,88 @@
 ---
 layout: post
 title: Precision with WinUI Rating control | Syncfusion
-description: Learn here all about precision mode defines the accuracy level and it has full, half and exact options of the SfRating control.
+description: Learn here all about the precision feature in the Rating control to set the accuracy level such as full, half, and exact.
```

Comment:
description: Learn all about the precision feature in the Rating control to set the accuracy level such as full, half, and exact here.

Reply:
Changed the content as mentioned above.